pages/common/nix-env.md

```diff
@@ -2,13 +2,17 @@
 
 > Manipulate or query Nix user environments.
 
-- Show available package with name or without name:
+- List installed packages:
```

Comment:
Perhaps phrasing this as "List all installed packages" will tie better with the next example, so that this one can more easily be mentally shifted towards the correct meaning (as in "oh, so that was also a query, but with no query terms, so it returns all results"). I feel that the "all" can help in gently nudging the brain towards this interpretation. Just a thought — WDYT?

Reply:
I agree, good catch. I'll make that change right now